ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/config/TlsClientOptions.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.config;
+
+public interface TlsClientOptions extends PkcsStoreConfig {
+  DownstreamTrustOptions getServerAuthConstraints();
```

Comment:
nit: it seems odd to extend an interface that was only being used in one place other than test code...

Reply:
Yup - this feels like a "has a", not an "is a" - so maybe use composition?

Reply:
The idea was to utilize/inherit the common method signatures from super interface. TLSClientOption was using keystore and keystore-password-file so does PkcsStoreConfig interface

Reply:
this interface is gone as it wasn't required at all.